src/odemis/acq/move.py

```diff
@@ -503,6 +503,7 @@ def _doCryoSwitchSamplePosition(self, future, target):
                 # The current mode doesn't change. Only X/Y/Z should move (typically
                 # only X/Y).
```

Comment:
This comment now disagree with the code. Can you adjust it to explain why we also move rx&rz? 
If I understand correctly, that's to handle the cases when the current rx/rz are not correct compared to the current values, for instance when in LOADING. However, I don't think that was never supposed to happen in the first place. My understanding was that this move can only happen if either in SEM of FM imaging. 

So I think that maybe instead of implementing this, we should raise an error if the current position is neither SEM nor FM imaging. That might also require some adjustments of the GUI, to make sure it's not possible to press the GRID buttons when neither SEM or FM is active. 
Note that I've been thinking of further refactoring the code to separate the change of posture with the change of grid. Such change would maybe help to avoid this issue... but I don't think it's necessary to do that now, unless you feel super motivated.

Reply:
Indeed the gui would require additional changes to implement the idea from this comment. Maybe I can just edit this comment for now?

Reply:
Yes, let's do so for now. Add a comment, with a `TODO: probably a better way would be to forbid grid switch if not in SEM/FM imaging posture".

Reply:
Please, still update this comment explaining why rx/rz are used while GRID 1&2 should always have the same rx/rz.